.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on: [push, pull_request]
+jobs:
+   arch:
+      runs-on: ubuntu-latest
+      container: archlinux:latest
+      steps:
+         - run: pacman --noconfirm --noprogressbar -Syyu
+         - run: pacman --noconfirm --noprogressbar -Sy make go asciidoc
```

Comment:
`pacman -S` should be used given the `-Syyu` on the previous line which can be `-Syu`.

Reply:
Fixed.